static/js/components/ObservationPlanRequestForm.jsx

```diff
@@ -35,18 +38,210 @@ const useStyles = makeStyles(() => ({
   localizationSelect: {
     width: "100%",
   },
+  fieldsToUseSelect: {
+    width: "80%",
+  },
   allocationSelectItem: {
     whiteSpace: "break-spaces",
   },
   localizationSelectItem: {
     whiteSpace: "break-spaces",
   },
+  fieldsToUseSelectItem: {
+    whiteSpace: "break-spaces",
+  },
+  SelectItem: {
+    whiteSpace: "break-spaces",
+  },
   container: {
     width: "99%",
     marginBottom: "1rem",
   },
 }));
 
+const FieldSelect = ({
+  skymapInstrument,
+  selectedFields,
+  setSelectedFields,
+}) => {
+  const classes = useStyles();
+
+  const handleSelectedFieldChange = (e) => {
+    setSelectedFields(e.target.value);
+  };
+
+  const clearSelectedFields = () => {
+    setSelectedFields([]);
+  };
+
+  const fields = [];
+  skymapInstrument?.fields?.forEach((field) => {
+    fields.push(Number(field.id));
+  });
+  fields.sort((a, b) => a - b);
+
+  return (
+    <div>
+      <InputLabel id="fieldsToUseSelectLabel">Fields to use</InputLabel>
+      <Select
+        inputProps={{ MenuProps: { disableScrollLock: true } }}
+        labelId="fieldsToSelectLabel"
+        name="fieldsToUseSelect"
+        className={classes.fieldsToUseSelect}
+        multiple
+        value={selectedFields || []}
+        onChange={handleSelectedFieldChange}
+      >
+        {fields?.map((field) => (
+          <MenuItem
+            value={field}
+            key={field}
+            className={classes.fieldsToUseSelectItem}
+          >
+            {field}
+          </MenuItem>
+        ))}
+      </Select>
+      <Button
+        id="clear-fieldsToUseSelect"
+        size="small"
+        color="secondary"
+        onClick={() => clearSelectedFields()}
+      >
+        Clear Fields
+      </Button>
```

Comment:
would be nice if this button color was a little darker. Actually a lot of the buttons on this page look different from the rest of the app, would be good if they were all unified. But maybe that can wait till the shift to mui5

Reply:
Yes this is exactly the kind of thing that will change there, so we can prettify the page once that goes in.